tools/tertiary-analysis/scanpy/scanpy-run-umap.xml

```diff
@@ -62,26 +58,13 @@ PYTHONIOENCODING=utf-8 scanpy-run-umap.py
           <option value="random">random</option>
         </param>
         <param name="maxiter" argument="--maxiter" type="integer" optional="true" label="Number of iterations of optimisation"/>
-        <param name="a" argument="-a" type="float" optional="true" label="More specific parameter controlling embedding, automatically determined from --min-dist and --spread if unset"/>
-        <param name="b" argument="-b" type="float" optional="true" label="More specific parameter controlling embedding, automatically determined from --min-dist and --spread if unset"/>
-        <param name="random_seed" argument="--random-seed" type="integer" value="0" label="Seed for numpy random number generator"/>
+        <param name="random_seed" argument="--random-state" type="integer" value="0" label="Seed for numpy random number generator"/>
       </when>
     </conditional>
-    <conditional name="do_plotting">
-      <param name="plot" type="boolean" checked="false" label="Make UMAP plot"/>
-      <when value="true">
-        <expand macro="output_plot_params"/>
-        <param name="color_by" argument="--color-by" type="text" value="louvain" label="Color by attributes, comma separated strings"/>
-      </when>
-      <when value="false"/>
-    </conditional>
   </inputs>
 
   <outputs>
     <data name="output_h5" format="h5" from_work_dir="output.h5" label="${tool.name} on ${on_string}: UMAP object"/>
-    <data name="output_png" format="png" from_work_dir="output.png" label="${tool.name} on ${on_string}: UMAP plot">
```

Comment:
same here, for methods like tSNE or UMAP, I would expect an actual plot being generated.